app/routers/session_answers.py

```diff
@@ -18,26 +20,34 @@ async def update_session_answer_in_a_session(
     session_id - the id of the session
     position_index - the position index of the session answer in the session answers array. This corresponds to the position of the question in the quiz
     """
+    logger.info(
+        f"Updating session answer for session: {session_id} at position: {position_index}. The answer is {session_answer.answer}"
```

Comment:
is session_answer.answer accessible here before doing jsonable_encoder(session_answer)

Reply:
also, sometimes, session_answer.answer won't change. but only `session_answer.visited` is set to True. we can add this also to logger

Reply:
For the first comment, yes we can access it